### PR TITLE
Added Tow 12 ( LV FightClub ) in Tow.pwn

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by ghost** [comment]
Russell u can explain that ? Segmentation fault (core dumped)

File: pawn/Elements/Player/Tow.pwn [hunk 1]
```diff
@@ -56,6 +57,7 @@ new towNames[NumberOfTowLocations][] =
     "Los Santos airport",
     "Main bank",
     "Train station"
```
Comment (RussellLVP) [inline pawn/Elements/Player/Tow.pwn]: You're missing a comma here too.

File: pawn/Elements/Player/Tow.pwn [hunk 2]
```diff
@@ -39,6 +39,7 @@ new Float:towLocations[NumberOfTowLocations][] =
     {1890.2701,     -2627.1948,     13.2740,        0.2716}, // LS Airport
     {2422.0320,     1113.8469,      10.4708,        0.3177}, // LV Main bank
     {2765.3484,     1281.4395,      10.4771,        269.4123} // LV Train Station
```
Comment (RussellLVP) [inline pawn/Elements/Player/Tow.pwn]: You're missing a comma here.

**Comment by ghost** [comment]
Thanks !

**Comment by ghost** [comment]
Russell Now it's ok 

u can check it if Good u can merge it ? :D
after that Help me for build a BETA Server in home :)

**Comment by RussellLVP** [comment]
This seems fine to me, thank you! :)

**Comment by ghost** [comment]
:dancer: I'm Very Happy :D

**Pull request merged**